docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -37,7 +37,7 @@ Make sure you have `prepare` goal of `quarkus-maven-plugin` enabled in your `pom
             <executions>
                 <execution>
                     <goals>
-                        <goal>prepare</goal>
+                        <goal>generate-code</goal>
```

Comment:
Why isn't `<goal>generate-code-tests</goal>` added in the example too?

Reply:
:+1: